Documentation/ceph-upgrade.md

```diff
@@ -234,50 +235,20 @@ time without compatibility support and without prior notice.
 
 Let's get started!
 
-### 1. Update modified permissions
-
-> **IMPORTANT**: Ensure that you are using the latest manifests from the `release-1.1` branch. If you
+> **IMPORTANT**: Ensure that you are using the latest manifests from the `release-1.2` branch. If you
 > have custom configuration options set in your 1.0 manifests, you will need to also alter those
 > values in the 1.1 manifests.
 
-A few permissions have been added in v1.1. To make updating these resources easy, special upgrade
-manifests have been created.
-
-Replace the namespace names in the new resources:
-
-```console
-sed "s/ROOK_SYSTEM_NAMESPACE/$ROOK_SYSTEM_NAMESPACE/g" upgrade-from-v1.0-create.yaml > upgrade-from-v1.0-create.yaml.tmp
-sed "s/ROOK_NAMESPACE/$ROOK_NAMESPACE/g" upgrade-from-v1.0-create.yaml.tmp > upgrade-from-v1.0-create.yaml
-rm -f upgrade-from-v1.0-create.yaml.tmp
-sed -i "s/ROOK_SYSTEM_NAMESPACE/$ROOK_SYSTEM_NAMESPACE/g" upgrade-from-v1.0-apply.yaml
-```
-
-If you have a v1.0 cluster running with CSI drivers enabled, delete the rbac rules created for CSI
-
-```console
-kubectl -n $ROOK_SYSTEM_NAMESPACE delete clusterrole.rbac.authorization.k8s.io/rbd-external-provisioner-runner-rules
-kubectl -n $ROOK_SYSTEM_NAMESPACE delete clusterrole.rbac.authorization.k8s.io/cephfs-external-provisioner-runner-rules
-```
-
-Apply the new permissions:
-
-```console
-kubectl create -f upgrade-from-v1.0-create.yaml
-kubectl apply -f upgrade-from-v1.0-apply.yaml
-```
-
-> **NOTE**: Ignore `Error from server (AlreadyExists)` error while running `kubectl create -f upgrade-from-v1.0-create.yaml`
+### 1. Update CSI Driver settings (if applicable)
```

Comment:
We don't need this section on updating the csi driver settings anymore, do we? Since the default in 1.1 is to enable it and the recommended csi images aren't different from what is in v1.1, the operator should automatically take care of any csi updates.

Reply:
I thought that, but users who set custom images will need to update them in the CRD. I think we will need this section for the remaining future.

Reply:
Why is this required for the rook upgrade? I would think they would want to update this when a new version of the csi driver is released, rather than doing it at the same time as the operator. 

Perhaps what we need is an appendix to the upgrade guide for things to optionally consider during upgrade. This seems like one of those things. I'd like to keep the upgrade flow for majority of users as simple as possible. Majority of users should just be using the csi defaults.

Reply:
An appendix seems wise. Or something like it? We could have a CSI upgrade guide section just like we have a Ceph guide section?